package.json

```diff
@@ -1,22 +1,26 @@
 {
   "name": "@inlivedev/inlive-js-sdk",
-  "version": "1.0.0",
-  "description": "",
+  "version": "0.1.0",
+  "description": "InLive JavaScript SDK",
   "keywords": [],
   "license": "MIT",
-  "source": "./packages/index.js",
-  "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
   "type": "module",
+  "exports": {
+    ".": "./packages/index.js",
+    "./app": "./packages/app/index.js",
+    "./stream": "./packages/stream/index.js",
+    "./event": "./packages/event/index.js",
+    "./packages/internal/*": null
+  },
```

Comment:
These exports replaced legacy `main` and `module` entry points. We can have multiple entry points for each module. This also restricts user that wants to access internal modules